### PR TITLE
Add file Upload Status Update 

### PR DESCRIPTION
**Pull request by FledgeXu**
# Goal:
Fix: #36 
## Abstract:
This PR allows WebUI, to detect the status of the file every so often, and fixes the issue where file upload errors with wrong marks.

**Notice: This PR doesn't fix the #56, that PR needs more discuss.**

### CODE REVIEW

File: frontend/public/environ.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "NAUTILUS_WEB_API": "http://localhost:8080/v1",
-  "NAUTILUS_FILE_QUOTA": "100000000",
-  "NAUTILUS_PROJECT_QUOTA": "100000000"
+  "NAUTILUS_FILE_QUOTA": 100000000,
+  "NAUTILUS_PROJECT_QUOTA": 100000000,
+  "NAUTILUS_REFRESH_TIME": 1000
```
Comment (rgaudin) [inline frontend/public/environ.json:5]: Unclear. `NAUTILUS_FILE_REFRESH_EVERY_MS` is better. Size doesn't matter here

File: frontend/src/components/FileTableRowComponent.vue [hunk 2]
```diff
@@ -49,12 +44,12 @@
       >
         <font-awesome-icon class="text-danger fs-5" :icon="['fas', 'xmark']" />
       </span>
-      <span v-else>
-        <font-awesome-icon class="text-primary fs-5" :icon="['fas', 'check']" />
-      </span>
+      <div v-else class="spinner-border text-secondary" role="status">
+        <span class="visually-hidden">Loading...</span>
```
Comment (rgaudin) [inline frontend/src/components/FileTableRowComponent.vue:48]: We've lost the progress indicator btw ; this is problematic. Uploading 100MB on a slow line is **very** long and user needs to see progress. Once it's _uploaded_ (but not in S3), we can switch the number for _Processing_

**Review by rgaudin — COMMENTED** [review]
Please rebase off main branch, it's missing some stuff.
I'll finish review tom. as the session issue prevents me from testing

**Comment by FledgeXu** [comment]
> Please rebase off main branch, it's missing some stuff.

I tried to rebase, but git suggests that it is updated.

**Review by rgaudin — APPROVED** [review]
Thank you ; We just need the upload progress (percentage for instance) and we can merge.
I've pushed a fix to #56 by using another session. Good enough for now, we can revisit later.

**Pull request merged**